includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -1446,11 +1446,32 @@ public function build_display_items( $itemized_display_items = false ) {
 			];
 		}
 
+		$calculated_total = apply_filters_deprecated(
+			'woocommerce_stripe_calculated_total',
+			[ WC_Stripe_Helper::get_stripe_amount( $order_total ), $order_total, WC()->cart ],
+			'9.6.0',
+			'wc_stripe_calculated_total',
+			'The woocommerce_stripe_calculated_total filter is deprecated since WooCommerce Stripe Gateway 9.6.0, and will be removed in a future version. Use wc_stripe_calculated_total instead.'
+		);
+
+		/**
+		 * Filters the calculated total for the order.
+		 *
+		 * @since 9.6.0
+		 *
+		 * @param float $calculated_total The calculated total from the deprecated filter.
+		 * @param float $order_total The order total.
+		 * @param WC_Cart $cart The cart object.
+		 *
+		 * @return float The calculated total.
+		 */
+		$calculated_total = apply_filters( 'wc_stripe_calculated_total', $calculated_total, $order_total, WC()->cart );
```

Comment:
As in the other cases, I think it would be simpler to initialise the variable outside the filter, and to avoid documenting the deprecated filter.
```suggestion
		$calculated_total = WC_Stripe_Helper::get_stripe_amount( $order_total );

		$calculated_total = apply_filters_deprecated(
			'woocommerce_stripe_calculated_total',
			[ $calculated_total, $order_total, WC()->cart ],
			'9.6.0',
			'wc_stripe_calculated_total',
			'The woocommerce_stripe_calculated_total filter is deprecated since WooCommerce Stripe Gateway 9.6.0, and will be removed in a future version. Use wc_stripe_calculated_total instead.'
		);

		/**
		 * Filters the calculated total for the order.
		 *
		 * @since 9.6.0
		 *
		 * @param float $calculated_total The calculated total.
		 * @param float $order_total The order total.
		 * @param WC_Cart $cart The cart object.
		 *
		 * @return float The calculated total.
		 */
		$calculated_total = apply_filters( 'wc_stripe_calculated_total', $calculated_total, $order_total, WC()->cart );
```